crates/uv-resolver/src/lock/requirements_txt.rs

```diff
@@ -312,6 +314,27 @@ impl<'lock> RequirementsTxtExport<'lock> {
             .map(|(index, package)| Requirement {
                 package,
                 marker: reachability.remove(&index).unwrap_or_default(),
+                dependents: graph
+                    .edges_directed(index, Direction::Incoming)
+                    .sorted_by_key(|edge| {
```

Comment:
What is the purpose of this sort?

Reply:
The purpose is to put on top any dependents that have the root as a direct dependent. This being the current project.

This is before sorting:
<img width="385" alt="image" src="https://github.com/user-attachments/assets/61cfa161-bfc8-4f5d-adb4-cccc6f3d7e6d" />

And this is after:
<img width="385" alt="image" src="https://github.com/user-attachments/assets/b4939efd-ee13-4fe0-a815-08f55005f6fa" />

I will add a secondary sort by name

Reply:
Personally I think I'd prefer to sort by name -- it's a little easier to parse visually since it's predictable.

Reply:
Do you mean just by name or first the project and then the rest of the dependents by name?

I tried to be consistent with the output of `uv pip compile`.

Reply:
I was proposing sorting _just_ by the dependent name. Doesn't `uv pip compile` just sort by name?

```rust
let mut dependents = graph
    .edges_directed(index, Direction::Incoming)
    .map(|edge| &graph[edge.source()])
    .map(uv_distribution_types::Name::name)
    .collect::<Vec<_>>();
dependents.sort_unstable();
dependents.dedup();
dependents
```